app/models/user.rb

```diff
@@ -21,4 +21,8 @@ class User < ApplicationRecord
   def three_most_recent_posts
     posts.order(created_at: :desc).limit(3)
   end
+
+  def admin?
+    role == 'admin'
+  end
```

Comment:
- [ ] Good Job for creating  their` user.admin?` method inside the user model kudos 👍 💯 .

Reply:
@Abenezer-Tilahun Thank you for your awesome review and wonderful feedback, I appreciate